doc/ref/peer.rst

```diff
@@ -97,6 +98,45 @@ to the manage and jobs runner functions.
         - jobs.*
 
 .. note::
+
+    Functions are matched using regular expressions.
+
+.. _peer-wheel:
+
+Peer Wheel Communication
+=========================
+
+.. versionadded: Fluorine
+
+Configuration to allow minions to execute wheel functions from the master is
+done via the ``peer_wheel`` option on the master. The ``peer_wheel``
+configuration follows the same logic as the ``peer`` option. The only
+difference is that access is granted to wheel modules.
+
+To open up access to all minions to all wheel:
+
+.. code-block:: yaml
+
+    peer_wheel:
+      .*:
+        - .*
+
+.. warning::
+
+    The above configuration allows all the minions to control the master,
```

Comment:
I have to be particular about the way this is phrased. What this says is that *this* configuration allows all minions to control the configuration files of the master. I think we need to be even more clear that _any_ minion that is allowed to control wheel functions on the master is effectively being *fully trusted* by the master.

Reply:
I tried to make the warning better.